scripts/client.js

```diff
@@ -0,0 +1,25 @@
+import KintoClient from "kinto-client";
+
+
+let client;
+
+export function setupClient({server, username, password}) {
```

Comment:
I wonder if we shouldn't pass the authorization header directly to later supports other authorization methods.